addons/account/tests/common.py

```diff
@@ -634,167 +634,14 @@ def setUpUsers(cls):
-
-        user.write({
-            'company_ids': [(6, 0, (cls.company_data['company'] + cls.company_data_2['company']).ids)],
-            'company_id': cls.company_data['company'].id,
-        })
-
-        cls.currency_data = cls.setup_multi_currency_data()
-
-        # ==== Taxes ====
-        cls.tax_sale_a = cls.company_data['default_tax_sale']
-        cls.tax_sale_b = cls.company_data['default_tax_sale'].copy()
-        cls.tax_purchase_a = cls.company_data['default_tax_purchase']
-        cls.tax_purchase_b = cls.company_data['default_tax_purchase'].copy()
-        cls.tax_armageddon = cls.setup_armageddon_tax('complex_tax', cls.company_data)
-
-        # ==== Products ====
-        cls.product_a = cls.env['product.product'].create({
-            'name': 'product_a',
-            'uom_id': cls.env.ref('uom.product_uom_unit').id,
-            'lst_price': 1000.0,
-            'standard_price': 800.0,
-            'property_account_income_id': cls.company_data['default_account_revenue'].id,
-            'property_account_expense_id': cls.company_data['default_account_expense'].id,
-            'taxes_id': [(6, 0, cls.tax_sale_a.ids)],
-            'supplier_taxes_id': [(6, 0, cls.tax_purchase_a.ids)],
-        })
-        cls.product_b = cls.env['product.product'].create({
-            'name': 'product_b',
-            'uom_id': cls.env.ref('uom.product_uom_dozen').id,
-            'lst_price': 200.0,
-            'standard_price': 160.0,
-            'property_account_income_id': cls.copy_account(cls.company_data['default_account_revenue']).id,
-            'property_account_expense_id': cls.copy_account(cls.company_data['default_account_expense']).id,
-            'taxes_id': [(6, 0, (cls.tax_sale_a + cls.tax_sale_b).ids)],
-            'supplier_taxes_id': [(6, 0, (cls.tax_purchase_a + cls.tax_purchase_b).ids)],
-        })
-
-        # ==== Fiscal positions ====
-        cls.fiscal_pos_a = cls.env['account.fiscal.position'].create({
-            'name': 'fiscal_pos_a',
-            'tax_ids': [
-                (0, None, {
-                    'tax_src_id': cls.tax_sale_a.id,
-                    'tax_dest_id': cls.tax_sale_b.id,
-                }),
-                (0, None, {
-                    'tax_src_id': cls.tax_purchase_a.id,
-                    'tax_dest_id': cls.tax_purchase_b.id,
-                }),
-            ],
-            'account_ids': [
-                (0, None, {
-                    'account_src_id': cls.product_a.property_account_income_id.id,
-                    'account_dest_id': cls.product_b.property_account_income_id.id,
-                }),
-                (0, None, {
-                    'account_src_id': cls.product_a.property_account_expense_id.id,
-                    'account_dest_id': cls.product_b.property_account_expense_id.id,
-                }),
-            ],
-        })
+class AccountTestSetup:
```

Comment:
the file already contains a lot of classes. Please docstring (and/or cleanup now or later) to document when to use it and what utilities it offers